docs/reference/commandline/tag.md

```diff
@@ -29,7 +29,7 @@ periods and dashes. A tag name may not start with a period or a dash and may
 contain a maximum of 128 characters.
 
 You can group your images together using names and tags, and then upload them
-to [*Share Images via Repositories*](../../tutorials/dockerrepos.md#contributing-to-docker-hub).
+to [*Share Images via Repositories*](https://docs.docker.com/docker-hub/repos/).
```

Comment:
Looks like this should have been https://docs.docker.com/engine/tutorials/dockerrepos/#/contributing-to-docker-hub

@yuexiao-wang are you interested in doing a follow up?